client/src/app/site/motions/services/motion-repository.service.ts

```diff
@@ -152,11 +152,23 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
      * Set the recommenders state of a motion
      *
      * @param viewMotion target motion
-     * @param stateId the number that indicates the state
+     * @param recommendationId the number that indicates the recommendation
      */
-    public async setRecommenderState(viewMotion: ViewMotion, stateId: number): Promise<void> {
+    public async setRecommendation(viewMotion: ViewMotion, recommendationId: number): Promise<void> {
         const restPath = `/rest/motions/motion/${viewMotion.id}/set_recommendation/`;
-        await this.httpService.put(restPath, { recommendation: stateId });
+        await this.httpService.put(restPath, { recommendation: recommendationId });
+    }
+
+    /**
+     * Set the state of a motion
+     *
+     * @param viewMotion target motion
+     * @param stateId the number that indicates the state
+     */
+    public async setCatetory(viewMotion: ViewMotion, categoryId: number): Promise<void> {
+        const motion = viewMotion.motion;
+        motion.category_id = categoryId;
+        await this.update(motion, viewMotion);
```

Comment:
of this function should be executed on multiple motions, that "might" need a
`return await` so synchronize the execution of the loops